daemon/cluster/executor/container/controller.go

```diff
@@ -142,7 +142,68 @@ func (r *controller) Start(ctx context.Context) error {
 		return errors.Wrap(err, "starting container failed")
 	}
 
-	return nil
+	// no health check
+	if ctnr.Config == nil || ctnr.Config.Healthcheck == nil {
+		return nil
+	}
+
+	healthCmd := ctnr.Config.Healthcheck.Test
+
+	if len(healthCmd) == 0 {
+		// this field should be filled, even if inherited from image
+		// if it's empty, health check will always be at starting status
+		// so treat it as no health check, and return directly
+		return nil
+	}
+
+	// health check is disabled
+	if healthCmd[0] == "NONE" {
```

Comment:
This API is unfortunate... why does `NONE` mean no check? What is wrong with empty?

cc @crosbymichael @talex5